terrautils/sensors.py

```diff
@@ -118,75 +121,94 @@
               "fixed_metadata_datasetid": "TBD"
         },
 
-        'rgb_fullfield': {
+        'fullfield': {
+            'display': 'Full Field Stitched Mosaics',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{filename}',
             'pattern': '{sensor}_lv1_{station}_{date}{opts}.tif',
         },
 
-        'flirIrCamera': {
+        'hyperspectral2nc_vnir': {
+            'display': 'VNIR Hyperspectral NetCDFs',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{time}/{filename}',
-            'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.tif',
+            'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.nc',
+        },
+
+        'hyperspectral2nc_swir': {
+            'display': 'SWIR Hyperspectral NetCDFs',
+            'template': '{base}/{station}/Level_1/'
+                        '{sensor}/{date}/{time}/{filename}',
+            'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.nc',
         },
 
         'stereoTop_geotiff': {
+            'display': 'stereoTop GeoTIFFs',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{time}/{filename}',
             'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.tif',
         },
 
         'stereoTop_canopyCover': {
+            'display': 'stereoTop Canopy Cover',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{time}/{filename}',
             'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.tif',
         },
 
         'texture_analysis': {
```

Comment:
I'll add RGB to the display names. The "R" was because R was underlying script of this extractor, but i suppose that isn't relevant to end users.